.test-infra/jenkins/job_PostCommit_Java_Nexmark_Dataflow.groovy

```diff
@@ -45,6 +45,7 @@ NoPhraseTriggeringPostCommitBuilder.postCommitJob('beam_PostCommit_Java_Nexmark_
               ' -Pnexmark.args="' +
```

Comment:
Most of these switches are same across file. You can move them into a constant and specify only different switches explicitly.

Reply:
There are several files like this. I can address it in another PR.